src/components/ImageWithSizeCalculation.tsx

```diff
@@ -37,14 +39,17 @@ type ImageWithSizeCalculationProps = {
  * performing some calculation on a network image after fetching dimensions so
  * it can be appropriately resized.
  */
-function ImageWithSizeCalculation({url, style, onMeasure, isAuthTokenRequired}: ImageWithSizeCalculationProps) {
+function ImageWithSizeCalculation({url, style, onMeasure, onLoadFailure, isAuthTokenRequired}: ImageWithSizeCalculationProps) {
     const styles = useThemeStyles();
     const isLoadedRef = useRef<boolean | null>(null);
     const [isImageCached, setIsImageCached] = useState(true);
     const [isLoading, setIsLoading] = useState(false);
 
+    const source = useMemo(() => ({uri: url}), [url]);
+
     const onError = () => {
         Log.hmmm('Unable to fetch image to calculate size', {url});
+        onLoadFailure?.();
```

Comment:
NAB: Should we provide default behavior to display a fallback image in this component?